backend.py

```diff
@@ -276,6 +276,12 @@ def get_tiles(self, coordinates):
             # Return hole tile.
             return [HoleTile()]
 
+    def active_robots(self, state):
```

Comment:
Jaký je rozdíl mezi `self` a `state`?

Reply:
To jsem se chtěla zeptat že nevím co tam má být za argumenty, v tom tápu a netuším.... a když na to koukám tak žádný, ale nevím co tam tedy má být, předpokládám že jen self? 
A pokud tu metodu volám jinde, tak použiju místo   
`for robot in state.active_robots(state):`  jenom `for robot in state.active_robots:`   ? Děkuju.

Reply:
Projdeme to na srazu :)